src/patternfly/components/DescriptionList/examples/DescriptionList.md

```diff
@@ -9,7 +9,12 @@ cssPrefix: pf-c-description-list
 
 ### Default
 ```hbs
-{{> description-list__example description-list--title="Default DL" description-list--header="Test"}}
+{{> description-list__example description-list--title="Default DL"}}
+```
+
+### Clickable term
+```hbs
+{{> description-list__example description-list--title="Clickable term" description-list-term--TextIsClickable="true"}}
```

Comment:
You could also just define `description-list-text--IsClickable="true"` here and you wouldn't need to set that var again on `{{#> description-list-text}}`

Reply:
Yes, that is true, but since we're using `__text` in both `__term` and `__description`, I had to isolate `__term > __text` from `__description > __text` so they're not all clickable.

Reply:
Ah, that makes sense!